test/feature/mac/parameters.yml

```diff
@@ -1,5 +1,5 @@
 # Receivers that agent needs to tests
-receivers: "host"
+receivers: ["host","statsd"]
```

Comment:
`Host` is instance system resources such as (e.g cpu,mem). We will add `emf` and `collectd` in the following future.